### PR TITLE
Ci updates

### PR DESCRIPTION
**Pull request by hrpatel**

### CODE REVIEW

**Comment by hrpatel** [comment]
Test

```bash
root@ip-10-100-12-101:/home/ubuntu/bashtester# ./run.sh env 2>/dev/null| grep -E "VERSION|LATEST"
_BASH_VERSION=3.0
_BASH_LATEST_PATCH=22
_BASH_VERSION=3.1
_BASH_LATEST_PATCH=23
_BASH_VERSION=3.2
_BASH_LATEST_PATCH=57
_BASH_VERSION=4.0
_BASH_LATEST_PATCH=44
_BASH_VERSION=4.1
_BASH_LATEST_PATCH=17
_BASH_VERSION=4.2
_BASH_LATEST_PATCH=53
_BASH_VERSION=4.3
_BASH_LATEST_PATCH=48
_BASH_VERSION=4.4
_BASH_LATEST_PATCH=23
_BASH_VERSION=5.0-beta
_BASH_LATEST_PATCH=0
```

**Comment by hrpatel** [comment]
Added more test results to the README as examples

**Comment by antevens** [comment]
+1

File: run.sh [hunk 1]
```diff
@@ -118,6 +116,6 @@ bash_images=( ${bash_images:-"${default_bash_images[@]}"} )
 
 for version in "${bash_images[@]}" ; do
     echo "########## ${version} ##########"
-    target_bash_version=${version} docker-compose run bash-tester "${@}"
+    target_bash_version=${version} docker-compose run bash-tester /usr/local/bin/bash -c "${@}"
```
Comment (adamgilbert) [inline run.sh]: Should ${version} be in quotes?

File: run.sh [hunk 2]
```diff
@@ -90,12 +90,10 @@ import_lib
...
 original_pwd="${PWD}"
-project_path="$(readlink -f "${BASH_SOURCE[0]}" 2> /dev/null || realpath "${BASH_SOURCE[0]}" 2> /dev/null || greadlink -f "${BASH_SOURCE[1]}" 2> /dev/null)"
+project_path="$(readlink -f "${BASH_SOURCE[0]}" 2> /dev/null || realpath "${BASH_SOURCE[0]}" 2> /dev/null || greadlink -f "${BASH_SOURCE[0]}" 2> /dev/null)"
 project_path="$(dirname ${full_path:-${0}})"
 cd "${project_path}" && add_on_exit "cd ${original_pwd}"
 
-dcc="docker-compose run bash-tester"
-
 if [ "${#}" -eq 0 ] || [ "${1}" == '--help' ] ; then
     color_echo cyan "Bash tester, tests a bash script/command on all currently supported bash versions"
```
Comment (adamgilbert) [inline run.sh]: I think we're doing single quotes on non-interpolated strings.

File: run.sh [hunk 3]
```diff
@@ -90,12 +90,10 @@ import_lib
...
     color_echo cyan "Typically this would be used as a git submodule and will map it's parent directory as /code"
```
Comment (adamgilbert) [inline run.sh:99]: Same here.

**Review by adamgilbert — CHANGES_REQUESTED** [review]
Minor style notes... +1 otherwise.

File: run.sh (hunk 2 above)
Comment (hrpatel) [reply run.sh]: Done

File: run.sh (hunk 3 above)
Comment (hrpatel) [reply run.sh:99]: There is a single quote in the line so I'm leaving the outside ones as is

File: run.sh (hunk 1 above)
Comment (hrpatel) [reply run.sh]: Done

**Review by adamgilbert — APPROVED** [review]

File: run.sh (hunk 3 above)
Comment (adamgilbert) [reply run.sh:99]: Works for me.

**Pull request merged**